src/features/user/state/ui/CouchDB.saga.ts

```diff
@@ -0,0 +1,27 @@
+import { takeLatest, select, call } from 'redux-saga/effects';
+import { getType } from 'typesafe-actions';
+import { syncSaga } from 'sagas/sync';
+import { loadSetting } from 'sagas/settings';
+import { getCouchDBSetting } from './CouchDB.selector';
+import * as CouchDB from './CouchDB.action';
+import { isUserLoggedIn } from '../User.saga';
+
+export function* finishCouchDBSettingSaga() {
+  const couchDB = yield select(getCouchDBSetting);
+  const userInfo = {
+    'accessToken': '',
+    "couchDB": { ...couchDB,
+      "transactions": couchDB.url + "/money_tracker_transactions_" + couchDB.username,
```

Comment:
let's use template literal instead here, e.g. 
```
"transactions": `${couchDB.url}/money_tracker_transactions_${couchDB.username}`
```

Reply:
also, any reason to change the prefix names for collections? can we keep it the same as hosted moneytracker collections, e.g. `${couchDB.url}/transactions_${couchDB.username}` etc

Reply:
The main reason is to allow other applications to use a single CouchDB server. Not sure if this is good or bad practice for using CouchDB.

I'll change this to keep it the same as hosted moneytrack collections.

Reply:
As I started experimenting more with CouchDB, I think it would be better to add a common prefix to CouchDB database names for moneytracker.

I'm changing the database name to `moneytracker_${couchDB.username}_tags` etc. This way databases belonging to the same user would be groupped together in CouchDB's web interface. This makes administration a little easier.

As it does not affect syncing to the hosted CouchDB, would you like this change?